src/main/webapp/app/text-assessment/text-assessment.component.ts

```diff
@@ -252,12 +258,14 @@ export class TextAssessmentComponent implements OnInit, OnDestroy, AfterViewInit
         this.formattedSampleSolution = this.artemisMarkdown.htmlForMarkdown(this.exercise.sampleSolution);
 
         this.result = this.participation.results[0];
-        this.hasComplaint = this.result.hasComplaint;
 
-        this.assessments = this.result.feedbacks || [];
+        const feedbacks = this.result.feedbacks || [];
+        const generalFeedbackIndex = feedbacks.findIndex(feedback => feedback.reference == null);
+        this.generalFeedback = feedbacks[generalFeedbackIndex] || new Feedback();
+        feedbacks.splice(generalFeedbackIndex, 1);
```

Comment:
When there is no general feedback (e.g. when loading an old assessment) `generalFeedbackIndex` will be -1 and the first feedback element will be removed by this line. It will not be shown to the tutor in the assessment view then.